src/libs/ReportUtils.js

```diff
@@ -465,27 +465,41 @@ function getIcons(report, personalDetails, policies, defaultIcon = null) {
     return _.map(sortedParticipants, item => item.avatar);
 }
 
+/**
+ * Gets the personal details for a login by looking in the allPersonalDetails from Onyx. If it doesn't exist in Onyx, then a default object is constructed.
+ * @param {String} login
+ * @returns {Object}
+ */
+function getPersonalDetailsForLogin(login) {
+    if (!login) {
+        return {};
+    }
+    return (allPersonalDetails && allPersonalDetails[login]) || {
+        login,
+        displayName: Str.removeSMSDomain(login),
+        avatar: getDefaultAvatar(login),
+    };
```

Comment:
I kinda like using `lodashGet` more than `&&` with `||`, what do you think of something like this?

```suggestion
    return lodashGet(allPersonalDetails, login, {
        login,
        displayName: Str.removeSMSDomain(login),
        avatar: getDefaultAvatar(login),
    });
```